azure-discovery/build.gradle

```diff
@@ -20,6 +20,7 @@ task writePropertiesFile {
 jar.dependsOn('writePropertiesFile')
 
 dependencies {
+    implementation project(':es:es-server')
```

Comment:
why is this new dependency needed?

Reply:
Before it was implicitly there due to the common dependency which depended on es-server and exposed it fully with compile